diff --git "a/c:\\xampp\\htdocs\\bundling2\\resources\\views\\layout\\layoutDashboard.blade.php" "b/c:\\xampp\\htdocs\\bundling2\\scratch_wrap_parents_2.blade.php"
index acc1281..05f29a0 100644
--- "a/c:\\xampp\\htdocs\\bundling2\\resources\\views\\layout\\layoutDashboard.blade.php"
+++ "b/c:\\xampp\\htdocs\\bundling2\\scratch_wrap_parents_2.blade.php"
@@ -497,6 +497,7 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                         {{-- MENU KEUANGAN --}}
                         <li class="nav-header user-panel"></li>
                         <li class="nav-header">Transaksi</li>
+@if(\App\Helpers\AksesHelper::cekAny(['/laporan-pengeluaran-harian', '/returObat', '/pembayaran-ralan', '/cari-piutang-ralan', '/cari-piutang-ranap', '/cari-bayar-piutang', '/bayar-piutang-khanza', '/cari-cob-bayar-piutang', '/cob-harian', '/piutang-harian', '/cari-bayar-umum', '/bayar-piutang-karyawan', '/invoice-asuransi', '/pasien-terdaftar']))
                         <li class="nav-item">{{-- menu-open --}}
                             <a href="#" class="nav-link">{{-- active --}}
                                 <i class="nav-icon fas fa-credit-card"></i>
@@ -620,7 +621,9 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
                         {{-- MENU DETAIL TINDAKAN --}}
+@if(\App\Helpers\AksesHelper::cekAny(['/ralan-dokter', '/ralan-paramedis', '/ralan-dokter-paramedis', '/operasi-and-vk', '/operasi-and-vk-kso', '/ranap-dokter', '/ranap-paramedis', '/ranap-dokter-paramedis', '/periksa-radiologi', '/periksalabpa', '/jm-asuransi']))
                         <li class="nav-item">{{-- menu-open --}}
                             <a href="#" class="nav-link">{{-- active --}}
                                 <i class="nav-icon fas fa-hospital-user"></i>
@@ -723,6 +726,8 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
+@if(\App\Helpers\AksesHelper::cekAny(['/ralan-dokter2', '/ralan-paramedis2', '/ralan-dokter-paramedis2', '/ranap-dokter4', '/ranap-paramedis2', '/ranap-dokter-paramedis2', '/periksa-radiologi2', '/operasi-and-vk1', '/periksalabpabulanan', '/jm-bpjs']))
                         <li class="nav-item">{{-- menu-open --}}
                             <a href="#" class="nav-link">{{-- active --}}
                                 <i class="nav-icon fas fa-hospital-user"></i>
@@ -817,8 +822,10 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
                         {{-- DETAIL TINDAKAN UMUM --}}
                         @if (session('edit_registrasi') === 'true')
+@if(\App\Helpers\AksesHelper::cekAny(['/ralan-dokter-umum', '/ralan-paramedis-umum', '/ralan-dokter-paramedis-umum', '/operasi-and-vk-umum', '/ranap-dokter-umum', '/ranap-paramedis-umum', '/ranap-dokter-paramedis-umum', '/periksa-radiologi-umum', '/periksalabpaumum', '/jm-umum']))
                             <li class="nav-item">{{-- menu-open --}}
                                 <a href="#" class="nav-link">{{-- active --}}
                                     <i class="nav-icon fas fa-hospital-user"></i>
@@ -913,8 +920,10 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                     </li>
                                 </ul>
                             </li>
+@endif
                         @endif
                                                 {{-- DETAIL TINDAKAN BPJS --}}
+@if(\App\Helpers\AksesHelper::cekAny(['/ralan-dokter-bpjs', '/ralan-paramedis-bpjs', '/ralan-dokter-paramedis-bpjs', '/ranap-dokter-bpjs', '/ranap-paramedis-bpjs', '/ranap-dokter-paramedis-bpjs', '/periksa-radiologi-bpjs', '/operasi-and-vk-bpjs', '/periksalabpabpjs', '/jm-bpjs']))
                         <li class="nav-item">{{-- menu-open --}}
                             <a href="#" class="nav-link">{{-- active --}}
                                 <i class="nav-icon fas fa-hospital-user"></i>
@@ -1009,9 +1018,11 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
                         {{-- MENU CASEMIX --}}
                         <li class="nav-header user-panel"></li>
                         <li class="nav-header">Gabung Berkas - Tools</li>
+@if(\App\Helpers\AksesHelper::cekAny(['/list-pasein-ralan', '/list-pasein-ralan2', '/list-pasein-ranap', '/list-pasein-ranap2', '/data-inacbg', '/casemix-home', '/bpjs/croscek-pasien-pulang', '/croscheck-coding', '/setting-bpjs-casemix']))
                         <li class="nav-item">{{-- menu-open --}}
                             <a href="#" class="nav-link">{{-- active --}}
                                 <i class="nav-icon fas fa-hospital"></i>
@@ -1095,7 +1106,9 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </ul>
                             @endif
                         </li>
+@endif
                         {{-- MENU FARMASI --}}
+@if(\App\Helpers\AksesHelper::cekAny(['list-pasien-farmasi', 'list-pasien-farmasi2', 'minimal-stok-obat']))
                         <li class="nav-item">{{-- menu-open --}}
                             <a href="#" class="nav-link">{{-- active --}}
                                 <i class="nav-icon fas fa-mortar-pestle"></i>
@@ -1131,7 +1144,9 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </ul>
                             @endif
                         </li>
+@endif
                         {{-- MENU BRIGING BPJS --}}
+@if(\App\Helpers\AksesHelper::cekAny(['pre-claim-validator', 'kirim-taskid-bpjs', 'kirim-taskid-bpjs2', 'update-jadwal-dokter', 'sep-vclaim', 'icare']))
                         <li class="nav-item">{{-- menu-open --}}
                             <a href="#" class="nav-link">{{-- active --}}
                                 <i class="nav-icon fas fa-file"></i>
@@ -1205,9 +1220,11 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
                         <li class="nav-header user-panel"></li>
                         <li class="nav-header">Pelayanan - RM</li>
                         {{-- MENU ANTRIAN --}}
+@if(\App\Helpers\AksesHelper::cekAny(['anjungan-mandiri', 'antrian-pendaftaran', 'antrian', 'antrian/tv', 'regperiksabilling', 'info-kamar-ruangan', 'info-kamar', 'info-kamar2', '/info-kamar3', 'info-kamar-baru', 'setting-antrian']))
                         <li class="nav-item">{{-- menu-open --}}
                             <a href="#" class="nav-link">{{-- active --}}
                                 <i class="nav-icon fas fa-user"></i>
@@ -1308,7 +1325,9 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                             </ul>
 
                         </li>
+@endif
                         {{-- ANTRIAN POLI --}}
+@if(\App\Helpers\AksesHelper::cekAny(['antrian-poli', 'setting-antrian-poli', 'jadwal-dokter']))
                         <li class="nav-item">{{-- menu-open --}}
                             <a href="#" class="nav-link">{{-- active --}}
                                 <i class="nav-icon fas fa-layer-group"></i>
@@ -1342,7 +1361,9 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
                         {{-- ANTRIAN FARMASI --}}
+@if(\App\Helpers\AksesHelper::cekAny(['antrian-farmasi', 'farmasi/antrian', 'antrian-farmasi/panggil', 'antrian-farmasi-baru', 'antrian-farmasi/display-v2', 'display-farmasi', '/waktu-tunggu-farmasi']))
                         <li class="nav-item">
                             <a href="#" class="nav-link">
                                 <i class="nav-icon fas fa-pills"></i>
@@ -1408,7 +1429,9 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
                             {{-- MENU RM --}}
+@if(\App\Helpers\AksesHelper::cekAny(['laporan-borlosetc', 'kroscek-pasien', 'kroscek-general-consent', 'laporan-pasien', 'rm/diagnosa', 'berkas-rm', 'waktu-tunggu-pasien-bayar', 'rawat-jalan', 'kunjungan-ralan', 'status-data-rm', 'pasien-pulang-ranap', 'pasien-ranap-igd', 'pasien-per-episode', 'tabulasi-igd', 'pasien-meninggal', 'jumlah-pasien', 'skrining-ispa', 'kodingan-rm']))
                         <li class="nav-item">{{-- menu-open --}}
                             <a href="#" class="nav-link">{{-- active --}}
                                 <i class="nav-icon fas fa-file"></i>
@@ -1554,7 +1577,9 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
                         {{-- Pasien --}}
+@if(\App\Helpers\AksesHelper::cekAny(['rawat-inap', 'infokamarinap', 'kamar-inap-dr']))
                         <li class="nav-item">
                             <a href="#" class="nav-link">
                                 <i class="nav-icon fas fa-procedures"></i>
@@ -1588,7 +1613,9 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
 
+@if(\App\Helpers\AksesHelper::cekAny(['inventaris-barang']))
                         <li class="nav-item">
                             <a href="#" class="nav-link">
                                 <i class="nav-icon fas fa-boxes-stacked"></i>
@@ -1606,7 +1633,9 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
 
+@if(\App\Helpers\AksesHelper::cekAny(['laboratorium']))
                         <li class="nav-item">
                             <a href="#" class="nav-link">
                                 <i class="nav-icon fas fa-microscope"></i>
@@ -1623,6 +1652,7 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
 
                        <li class="nav-item has-treeview">
                             <a href="#" class="nav-link">
@@ -1717,6 +1747,7 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                         </li>
                         <li class="nav-header user-panel"></li>
                         <li class="nav-header">Tools - Keperawatan</li>
+@if(\App\Helpers\AksesHelper::cekAny(['/gizi']))
                         <li class="nav-item">
                             <a href="#" class="nav-link">
                                 <i class="nav-icon fas fa-utensils"></i>
@@ -1734,6 +1765,8 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
+@if(\App\Helpers\AksesHelper::cekAny(['/urologi']))
                         <li class="nav-item">
                             <a href="#" class="nav-link">
                                 <i class="nav-icon fas fa-user-md"></i>
@@ -1752,7 +1785,9 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
 
+@if(\App\Helpers\AksesHelper::cekAny(['/skriningtbc', '/skrining-tbc']))
                         <li class="nav-item">
                             <a href="#" class="nav-link">
                                 <i class="nav-icon fas fa-lungs"></i>
@@ -1777,7 +1812,9 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
                         {{-- MENU KEPERAWATAN --}}
+@if(\App\Helpers\AksesHelper::cekAny(['home-keperawatan', '/pasien-lebih-dari-1']))
                         <li class="nav-item">{{-- menu-open --}}
                             <a href="#" class="nav-link">{{-- active --}}
                                 <i class="nav-icon fas fa-user-nurse"></i>
@@ -1804,6 +1841,8 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                             </ul>
                             
                         </li>
+@endif
+@if(\App\Helpers\AksesHelper::cekAny(['/fisioterapi', '/antrian-fisioterapi', '/display-fisioterapi']))
                         <li class="nav-item">
                             <a href="#" class="nav-link">
                                 <i class="nav-icon fas fa-diagnoses"></i>
@@ -1838,6 +1877,8 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
+@if(\App\Helpers\AksesHelper::cekAny(['/laporan-lab/anti-hiv']))
                         <li class="nav-item">
                             <a href="#" class="nav-link">
                                 <i class="nav-icon fas fa-flask"></i>
@@ -1855,6 +1896,8 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
+@if(\App\Helpers\AksesHelper::cekAny(['/pemesanan-farmasi', '/belanja']))
                         <li class="nav-item">
                             <a href="#" class="nav-link">
                                 <i class="nav-icon fas fa-file-invoice"></i>
@@ -1882,6 +1925,8 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
+@if(\App\Helpers\AksesHelper::cekAny(['pkpa.monitoring']))
                         <li class="nav-item">
                             <a href="#" class="nav-link">
                                 <i class="nav-icon fas fa-user-graduate"></i>
@@ -1900,6 +1945,8 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
+@if(\App\Helpers\AksesHelper::cekAny(['/listnama']))
                         <li class="nav-item">
                             <a href="#" class="nav-link">
                                 <i class="nav-icon fas fa-envelope-open-text"></i>
@@ -1917,6 +1964,7 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
 
                         {{-- MENU KASIR --}}
                         <li class="nav-header">Kasir</li>
@@ -1930,6 +1978,7 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                         {{-- MENU KEPEGAWAIAN --}}
                         <li class="nav-header user-panel"></li>
                         <li class="nav-header">Kepegawaian</li>
+@if(\App\Helpers\AksesHelper::cekAny(['/berkas-pegawai', 'berkas.pegawai.semua']))
                         <li class="nav-item">
                             <a href="#" class="nav-link">
                                 <i class="nav-icon fas fa-users-cog"></i>
@@ -1955,6 +2004,7 @@ class="brand-image img-circle elevation-3" style="opacity: 0.8" />
                                 </li>
                             </ul>
                         </li>
+@endif
 
                         <li class="nav-item mt-5">
                         </li>
